### ISSUE TITLE
FE | Profile | Phone # Accessibility Fix

### ISSUE THREAD

**Issue by SKasimow** [department-of-veterans-affairs/va.gov-team#44534]
## Background

In #38905, the following was discovered:

> I also found that the screen reader for the phone number is not that accessible. It reads out the phone number as a number, 123-123-1,234, would be read out as one billion, two hundred and thirty-one million, two hundred and thirty-one thousand two hundred and thirty-four. That should probably be fixed in a ticket as well.
Describe the issue background and scope

## Tasks
- [x] Fix the screen reader for phone numbers to read out as a phone number vs a general number.

## Acceptance Criteria
- [ ] Deploy fix to Staging behind feature flag
- [x] Notify PM that the change is ready to be validated on Staging
- [x] Deploy changes to Production once approved

**Comment by andaleliz** [comment]
@TiffanyPender do you agree this is a `508-defect-4`?

**Comment by adamwhitlock1** [comment]
For this issue I think it might be specific to the screen reader application. I investigated several things on my Mac with voice over activated:

- The standard 'data view' of the phone number reads out correctly to me, and reads each chunk of digits at a time (123-456-7890) 

- When editing a phone number, the input field is set to type="tel" so the input is correct as far as the HTML spec, but when I erased the whole number and started typing in a new phone number it was read as a long number (350 being read as three hundred fifty) BUT as I continued to type out the phone number it stopped reading it as a single long number and started reading the number one digit at a time ONCE I reached a number at least 7 numbers long.

- I investigated if there were any other HTML/CSS attributes that I could add to enforce speaking digit by digit no matter what, and I discovered that there is a CSS property called 'speak' that can be se to a value of 'digits' and that should provide a hint to screen readers that the element should be spoken in that way. Unfortunately setting the input to have that attribute had no effect on how Voice Over behaved.

- I also looked at the settings of Voice Over itself, and found that there is a setting for the application that allows a user to specify how numbers are spoken. The user menu in voice over has a Numbers menu which has a setting for character or digit pronunciations if set to digits, then the numbers will always be read one by one.

I guess I need some guidance as to whether there is something else that I should try or if this is just a limitation to screen readers and their individual quirks.

**Comment by TiffanyPender** [comment]
For screen reader consistency, it may be helpful to provide the screen reader equivalent in the updated aria-label. This includes adding spaces in between each number. For example, within the current Personal Information content area, this exists:

```html
<a href="tel:+1800-827-1000" aria-label="8 0 0 - 8 2 7 - 1 0 0 0.">800-827-1000</a>
```

**Comment by adamwhitlock1** [comment]
@TiffanyPender would an aria-label be an attribute that would be added to an input element specifically? 

The issue arises right now when a user is entering a new phone number into the input and has so far only a few numbers like '630' into the input then it is read as 'six hundred thirty' instead of 'six three zero'. I'm not sure how you would use an aria label dynamically like that as the user types, and if that would function as expected for the screen reader.

**Comment by TiffanyPender** [comment]
@adamwhitlock1 Ok, I think the issue I was referring to is something different - when links (anchor elements) to telephone numbers do not read out individual numbers. Including the numbers with proper spacing in the aria-label of a link would work for that situation.

In your scenario, I would avoid using an aria-label directly on an input element as there should be a `<label>` element present. Also, with the input type set to `tel`, a screen reader will typically only read the numbers individually if the text entered is formatted as a phone number. For example, if the input field contains 123-456-7890, it would read the individual numbers as "one two three....".

That may require some additional testing, along with the CSS "speak" properties.
